### ISSUE TITLE
Release 16 on pypi?

### ISSUE THREAD

**Issue by jpecar** [deeptools/HiCMatrix#45]
Hi,

it looks like latest available release on pypi is 15. Any chance to make 16 available there too? Thanks.

**Comment by boegel** [comment]
Same goes for version 17: it's tagged in GitHub, but not published through PyPI?

**Comment by lldelisle** [comment]
Yes sorry, we have some issues with the pypi deployment. @bgruening
